src/Process/Listener/MutationTestingResultsLoggerSubscriber.php

```diff
@@ -72,54 +82,63 @@ public function onMutationTestingFinished(MutationTestingFinished $event)
 
         $logTypes = $this->filterLogTypes($logTypes);
 
-        foreach ($logTypes as $logType => $directory) {
-            $this->useLogger($logType);
+        foreach ($logTypes as $logType => $config) {
+            $this->useLogger($logType, $config);
         }
     }
 
     private function filterLogTypes(array $logTypes): array
     {
         $allowedFileTypes = [
-            self::TEXT_FILE,
-            self::DEBUG_FILE,
-            self::SUMMARY_FILE,
+            ResultsLoggerTypes::TEXT_FILE,
```

Comment:
could you please create a new constant variable in the `ResultsLoggerTypes` which will have all of them?